hack/install-deps.sh

```diff
@@ -177,9 +177,8 @@ ${sudo} sh -c "PATH=${PATH} make install -e DESTDIR=${CONTAINERD_DIR}"
 #Install crictl
 checkout_repo ${CRITOOL_PKG} ${CRITOOL_VERSION}
 cd ${GOPATH}/src/${CRITOOL_PKG}
-make
-# make install critools require GOPATH to be set correctly.
-${sudo} make install -e BINDIR=${CRICTL_DIR} GOPATH=${GOPATH}
+make crictl
+${sudo} install -D -m 755 ${GOPATH}/bin/crictl ${CRICTL_DIR}/crictl
```

Comment:
Should we just add new build and install tags in critool's makefile so we don't have to do custom install?  Would also slim the build since this path is not installing critest no need to build it above.

Reply:
I can do that as next step.